### PR TITLE
[Rebase] Collapse memory related tracks by default

### PR DESCRIPTION
**Pull request by vickyliu-go4it**
Memory tracks and pagefault tracks take too much space. They should be
collapsed by default.

### CODE REVIEW

**Review by karupayun — COMMENTED** [review]
Hey! I have 2 comments:
1- This should be in the release :)
2- Shouldn't we instead SetCollapse all tracks to true by default and only expand the Thread Tracks?

**Comment by vickyliu-go4it** [comment]
> Hey! I have 2 comments:
> 1- This should be in the release :)
> 2- Shouldn't we instead SetCollapse all tracks to true by default and only expand the Thread Tracks?

I will merge to the release branch also! 

For the second point, I would like to also hear about the suggestion from @florian-kuebler

**Review by florian-kuebler — APPROVED** [review]
LGTM, thanks! 
Regarding the default value of `is_collapsed_`, I do not have a strong opinion. We had it as a constructor argument once, but at that point it was only the GpuTracks that were collapsed by default, so we decided to remove the constructor argument. So bringing back this argument could also be an option.

**Review by florian-kuebler — APPROVED** [review]
So basically to recap my thoughts.
For the UX side, someone should decide which tracks should be expanded by default (if any).

For code internals, we need to consider two cases.
1) Almost all tracks have the same default value, in that case, keep the default value and only call the setter in the exception (as it is right now).

2) If there is basically more than one or maybe two exceptions, we should have no default (internally), but set the default behaviour in the constructor.

In general, I'd say this can be done after this PR, so does not affect my approval.

**Comment by vickyliu-go4it** [comment]
We decided not to change the default value of `is_collapsed_` in the release fix. Let's have a better solution after the release! 😄

**Pull request merged**
